### PR TITLE
Fix test failure detection when dumps are enabled.

### PR DESCRIPTION
**Pull request by pgavlin**

### CODE REVIEW

**Comment by pgavlin** [comment]
@RussKeldorph PTAL

File: tests/runtest.cmd [hunk 1]
```diff
@@ -233,12 +233,13 @@ del %CORE_ROOT%\mscorlib.ni.dll
...
 set __BuildLogRootName=TestRunResults
 call :msbuild "%__ProjectFilesDir%\runtest.proj" /p:Runtests=true /clp:showcommandline
+set __errorlevel=%errorlevel%
 
 if "%__CollectDumps%"=="true" (
     python "%__DumplingHelperPath%" collect_dump %errorlevel% "%__CrashDumpFolder%" %__StartTime% "CoreCLR_Tests"
 )
 
-if errorlevel 1 (
+if %__errorlevel%==1 (
```
Comment (RussKeldorph) [inline tests/runtest.cmd]: Run "help if" at cmd prompt.  'if errorlevel' is a >= comparison.

Comment (RussKeldorph) [inline tests/runtest.cmd]: (...assuming it's possible for msbuild to return something greater than 1)

Comment (pgavlin) [inline tests/runtest.cmd]: fixed

**Pull request merged**